### DISCUSSION TITLE
Mesh Quality

### DISCUSSION THREAD

**Discussion by mattvers123** [Ifsttar/I-Simpa#356] [General]
Dear all, 
I was running some reverberation calculations and comparing to the real tests. There was a big gap (I am just testing the simulation not experienced yet). Wanted to make sure that the mesh is correct. When I checked the mesh, regardless of the radius to edge ratio (tetgen documentation says that the we should have finer mesh with lower radius-edge ratios), mesh remained the same. The mesh seems to be very big and therefore not filling the whole volume (with meshing option visible). Any recommendations on how to refine the mesh? THanks

**Comment by nicolas-f** [comment]
Hi,

To have smaller tetrahedron you have to reduce the value of volume constraint:
https://i-simpa-wiki.readthedocs.io/fr/latest/code_configuration_meshing.html#meshing

But mesh has no impact on computation results on spps as it is only used as an index structure.

Maybe you should look on materials properties of your walls, floor and ceiling.

**Reply by mattvers123** [reply]
Thanks for the reply. Tried that also, but no changes. (from 100 m3- to 0.01 m3). Our total volume is 6.44 m3 (alpha cabin). Thanks.

**Reply by nicolas-f** [reply]
By default you have the tetgen option -Y provided. It remove the adding of points on the boundary, you can remove this option.

If you don't find change maybe its because you change the parameter of "classical theory" and preview with spps ? 

If you could provide your project here maybe we can help, because its hard to guess what you have done

**Reply by mattvers123** [reply]
Sure.  Here is the link to the file:
https://drive.google.com/file/d/16oUHeYMIkn9JwwgInePbXwhJ1lKgTrM_/view?usp=share_link

I have also attached what I see as mesh. 
![mesh](https://user-images.githubusercontent.com/50327772/227497671-f40925ed-6ffe-4c30-8dca-947c3a7bdacc.png)

Thanks a lot. Regards.

**Comment by nicolas-f** [comment]
You have to check Constraint of volume, then put as small value

![image](https://user-images.githubusercontent.com/1382241/227499703-2fe5441b-2dbc-4abf-baaf-e1a855720459.png)

SPPS is particle tracing (close to raytracing), it looks like you are using SPPS as you would use finite element methods.

I don't think you are using the good numerical code for your project. Maybe @Picaut could explain why SPPS is not appropriate to emulate reverberation of sound into such small space (at least at low frequency).

**Reply by mattvers123** [reply]
Thanks a lot for the responses and guidance. I will check your response.
Alpha Cabins are standardized at 6.44 m3 and the operating range is in general 400-10 kHz. Up to Schroeder frequency (1250 Hz in our case), acoustic modes seem to effect the reverberation time whereas about this frequency modes combine and the behavior changes. So I have to check all 1/3 Octaves in between these ranges. It would be great if we can discuss further for further analysis.
Thanks again.
